CONTRIBUTING.md

```diff
@@ -210,11 +210,11 @@ For testing code locally, you will need a very basic setup. There are a few requ
 * [Node.js](https://nodejs.org/en/download/) Current LTS version
 * [Github](https://github.com) account
 
-Note: If you install Python on Windows using the MSI installer, it is highly recommended to check the "Add to path"-box during installation. If you have not done so, see if one of the answers to the StackOverflow post [Adding Python path on Windows 7](http://stackoverflow.com/questions/6318156/adding-python-path-on-windows-7) can help you - it should also work fine for later versions of Windows.
-
-Windows typically doesn't have the *make* tool installed. Windows users without *make* should look at the "detailed setup" section below.
-
-As an alternative to Windows, a cloud IDE such as [Cloud 9](https://c9.io) can be used for a relatively easier setup. If you take this route, please update to the latest Python version with the following. (This is to avoid `InsecurePlatformWarning` errors that arise when the default Python 2.7.6 is used).
+> Note: If you install Python on Windows using the MSI installer, it is highly recommended to check the "Add to path"-box during installation. If you have not done so, see if one of the answers to the StackOverflow post [Adding Python path on Windows 7](http://stackoverflow.com/questions/6318156/adding-python-path-on-windows-7) can help you - it should also work fine for later versions of Windows.
```

Comment:
Ah, yeah. I just thought it looked cool visually -- happy to merge in a PR if you'd like to change it back. :)